src/transforms/kotlin/typography.ts

```diff
@@ -25,7 +25,7 @@ export default {
       Object.entries(token.value).reduce((props, [propName, val]) => {
         let output = props;
         if (textStylePropertiesMapping[propName]) {
-          output += `\t\t${textStylePropertiesMapping[propName]} = `;
+          output += `        ${textStylePropertiesMapping[propName]} = `;
```

Comment:
Rather than hardcoding spaces, could we have a global constant like `const INDENT_LEVEL = "    ";` and change this as follow

```suggestion
          output += `${INDENT_LEVEL + INDENT_LEVEL + textStylePropertiesMapping[propName]} = `;
```

It conveys intent a bit better, and we get a chance to document that 4 spaces is actually what we expect in this context.

There are a couple more places in this PR that hardcode spaces, I won't comment on all of them, but you get the gist